plugins/support-helphub/inc/helphub-customroles/includes/class-helphub-custom-roles.php

```diff
@@ -0,0 +1,246 @@
+<?php
```

Comment:
I'd like to consolidate this into the first file, splitting it over multiple files doesn't seem as sensible in this scenario.